File: C++/z10/zadanie5.cpp
#include <fstream>
#include <iostream>
#include <string>
#include <filesystem>
using namespace std;
namespace fs = filesystem;
 
int main()
{
    fs::path p="C:\\Projekty\\C++\\z10";
    int sum=0;
        try
        {
            for(const auto & asda : fs::recursive_directory_iterator(p))
            {
                if(fs::is_regular_file(asda))
                    sum += fs::file_size(asda);
            }
        }
        catch(const std::exception& e)
        {
            cout << "Folder nie istnieje" << endl;
        }   
    std::cout << sum;
}

File: JavaScript/Lista3/zadanie5.js
function sum() {
    let sum=0;
    for (let i = 0; i < arguments.length; i++) {
        sum = sum + arguments[i];
    }
    return sum;
}
sum(1,2,3);// 6
console.log(sum(1,2,3))
sum(1,2,3,4,5);// 15